examples/language/gpt/train_gpt_demo.py

```diff
@@ -2,6 +2,7 @@
 from functools import partial
 from time import time
 
+# 1122
```

Comment:
Remove line 5, do the same for other weird comments as well.

Reply:
OK, done!